include/FockSpace/ONV.hpp

```diff
@@ -65,6 +69,12 @@ class ONV {
      */
     ONV(size_t K, size_t unsigned_representation);
 
+    /**
+     *  @param K                        the number of orbitals
+     *  @param N                        the number of electrons
+     */
+    ONV(GQCP::K_N_pair pair);
```

Comment:
I can see why you need a special constructor for this (accepting `K` and `N` would lead to the same constructor declaration as the one with the unsigned representation), but I don't think you should be documenting `K` and `N` in this constructor. Rather, I would skip the `using K_N_pair` and just make the constructor accept a tuple:
```suggestion
    ONV(std::tuple<size_t, size_t> pair);
```
and then document that the pair should be composed of `K` and `N`.

Anyways, maybe some time should be spent on clarifying the API for these constructors?

Maybe we could just delete the `ONV(size_t K, size_t unsigned_representation)` constructor, since we already have one with an extra argument: the number of electrons `N`?